pkgs/development/pharo/vm/build-vm.nix

```diff
@@ -92,6 +92,9 @@ stdenv.mkDerivation rec {
     LD_LIBRARY_PATH="\$LD_LIBRARY_PATH:$libs" exec $out/pharo "\$@"
     EOF
     chmod +x "$out/bin/${cmd}"
+    cp ${libgit2}/lib/libgit2.so.0.26.6 "$out/"
+    cp ${libgit2}/lib/libgit2.so.26 "$out/"
+    cp ${libgit2}/lib/libgit2.so "$out/"
```

Comment:
Would a symlink work here too?

Reply:
` ln -s ${libgit2}/lib/libgit2.so* "$out"`

Reply:
Why does `libgit2` have to be linked/copied and not added to `buildInputs`, unlike all the other libs?
Maybe add a comment.

Reply:
The explanation can be copied from the commit message.

Reply:
>  This is needed because Iceberg assumes it'll be using the libgit2 bundled by Pharo, which is expected to be in the same folder as the VM itself. Even if libgit2 dependency is resolved at runtime, Iceberg performs that check and would fail if the file is not where Iceberg expects it to be.